packages/preview/dmi-thesis/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "dmi-thesis"
```

Comment:
This name seems a bit general, we try to avoid canonical/generic names to avoid an unfair advantage when discovering packages. See our [naming rules](https://github.com/typst/packages/blob/main/docs/manifest.md#naming-rules) for more info.